src/components/Session.vue

```diff
@@ -61,50 +101,70 @@ export default {
   			user: UserService.getUser()
   		});
     });
+
+    // Offer the option to ask a question
+    setTimeout(() => {
+      if (
+        !UserService.getUser().isVolunteer && 
+        SessionService.getPartner() === undefined
+      ) {
+        this.showModal = true;
+      }
+    }, 86400000);
```

Comment:
@stackptr It is supposed to be just 10 mins, but I used this amount of time so that this frontend can be merged but not used, since the corresponding backend isn't done yet.

Why to merge then?... because this PR contains new components that will help me to refactor some parts of the frontend while I work on the bugs I'll be solving in the following weeks.